src/graphql/typeDefs.graphql

```diff
@@ -654,10 +654,19 @@ type MonsterAction {
   usage: Usage
 }
 
+type MonsterArmorClass {
+  type: String!
```

Comment:
Considering there's a pretty small finite number of possible types, I think an enum might be more appropriate than a string. The type would look like this:
```graphql
enum MonsterArmorClassType {
  DEX
  NATURAL
  ARMOR
  SPELL
  CONDITION
}
```

Then you'd just use `MonsterArmorClassType!` instead of `String!`. Also, this will mean you'll need to update the resolver to make the type uppercase.

Reply:
Should we do that with other types? I think we are missing that for other discriminators.

Reply:
I'm going to sanity check if the `enum` needs to be uppercase.

Reply:
It does not.

Reply:
I added the enum for this type but we can go back through and remove the uppercase for a lot of these.

Reply:
Whenever the uppercase enums get removed, do you think you'll need an announcement for that as well? I'm not sure if that will be a breaking change or not. Also, I wonder if that will have any implications for filters that are currently relying on enums.